src/models/propulsion/FGTurbine.cpp

```diff
@@ -69,7 +69,7 @@ FGTurbine::FGTurbine(FGFDMExec* exec, Element *el, int engine_number, struct Inp
   MaxN1 = MaxN2 = 100.0;
   Augmented = AugMethod = Injected = 0;
   BypassRatio = BleedDemand = 0.0;
-  IdleThrustLookup = MilThrustLookup = MaxThrustLookup = InjectionLookup = 0;
+  IdleThrustLookup = MilThrustLookup = MaxThrustLookup = InjectionLookup = TSFCLookup = ATSFCLookup = nullptr;
```

Comment:
I'd rather replace the `TSFC` and `ATSFC` members by `FGParameter*`
```suggestion
  IdleThrustLookup = MilThrustLookup = MaxThrustLookup = InjectionLookup = nullptr;
  TSFC = new FGRealValue(0.8);
  ATSFC = new FGRealValue(1.7);
```